apps/deploy-web/src/utils/sdl/data.ts

```diff
@@ -17,10 +17,17 @@ export const defaultHttpOptions = {
   nextTimeout: 60000
 };
 
+const defaultCredentials = {
+  host: 'docker.io',
+  username: '',
+  password: '',
+};
+
 export const defaultService: ServiceType = {
   id: nanoid(),
   title: "service-1",
   image: "",
+  credentials: defaultCredentials,
```

Comment:
This is throwing and error in the build as the `username` and `password` are required.